src/app/static/src/app/store/translations/locales.ts

```diff
@@ -72,6 +72,7 @@ export interface Translations {
     createReleaseTooltip: string,
     currentProject: string,
     custom: string,
+    dataExploration: string,
```

Comment:
Again, I don't think we need to show this in a select, so shouldn't need to translate it.